src/components/intro-page/intro-custom-content.tsx

```diff
@@ -1,46 +1,38 @@
 /*
- * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import React, { Fragment, useMemo } from 'react'
-import { useIntroPageContent } from './hooks/use-intro-page-content'
-import { useApplicationState } from '../../hooks/common/use-application-state'
+import React, { useEffect, useState } from 'react'
 import { WaitSpinner } from '../common/wait-spinner/wait-spinner'
 import { RenderIframe } from '../editor-page/renderer-pane/render-iframe'
 import { RendererType } from '../render-page/window-post-message-communicator/rendering-message'
+import { useTranslation } from 'react-i18next'
+import { fetchFrontPageContent } from './requests'
 
 /**
  * Fetches the content for the customizable part of the intro page and renders it.
  */
 export const IntroCustomContent: React.FC = () => {
-  const introPageContent = useIntroPageContent()
-  const rendererReady = useApplicationState((state) => state.rendererStatus.rendererReady)
+  const { t } = useTranslation()
+  const [content, setContent] = useState<string[] | undefined>(undefined)
 
-  const spinner = useMemo(() => {
-    if (!rendererReady && introPageContent !== undefined) {
-      return <WaitSpinner />
-    }
-  }, [introPageContent, rendererReady])
+  useEffect(() => {
+    fetchFrontPageContent()
+      .then((content) => setContent(content.split('\n')))
+      .catch(() => setContent(undefined))
```

Comment:
Shouldn't we use a different value than undefined, to indicate an error instead of showing a WaitSpinner forever?